actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -2349,6 +2352,7 @@ def label(method, text = nil, options = {}, &block)
       # To prevent this the helper generates an auxiliary hidden field before
       # the very check box. The hidden field has the same name and its
       # attributes mimic an unchecked check box.
+      # To disable the hidden field pass +include_hidden: false+ in the options hash.
```

Comment:
Also needs `<code>`:

```suggestion
      # To disable the hidden field pass <code>include_hidden: false</code> in the options hash.
```